### PR TITLE
Jk product popularity

### PR DESCRIPTION
**Pull request by juliakimchung**
# Description

Completed product-popularity chart with order-data, customer-data, product-data, line-item data
product-popularity chart displays on the command line 

# Related Ticket(s)
#15 
## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x ] I certify that all existing tests pass, tested on the cli , displays well as intended 

## Documentation
All doc strings attached

### CODE REVIEW

File: cli/product_cli.py [hunk 1]
```diff
@@ -89,14 +89,14 @@ def run(self):
...
 
-        # total_customers = self.get_total_num_of_customers()
-        # total_orders = self.get_total_num_of_orders()
+        total_customers = self.get_total_num_of_customers()
+        total_orders = self.get_total_num_of_orders()
 
-        print("{:<18}{:<11}{:<11}{:<15}".format("Product", "Order", "Customers", "Revenue") )
-        print("{:<18}{:<11}{:<11}{:<15}".format("Baby Powder", 3, 2, "$96.00"))
-        print ( "{:*^55}".format("*"))
+        # print("{:<18}{:<11}{:<11}{:<15}".format("Product", "Order", "Customers", "Revenue") )
```
Comment (samphillips1879) [inline cli/product_cli.py]: This commented code should be removed.

Comment (juliakimchung) [reply cli/product_cli.py]: Good catch!!

**Comment by samphillips1879** [comment]
You need to modify the product-popularity view to account for the following requirements:
1. The product column must be 18 characters wide, and will display a maximum of 17 characters for the product name.
2. The orders column must be 11 characters wide.
3. The customers column must be 11 characters wide.
4. The revenue column must be 15 characters wide.

The way it's written now, the "coconut oil shampoo" product name is stretching that row, throwing off the view's allignment

**Comment by pinkston4** [comment]
👍

**Review by samphillips1879 — COMMENTED** [review]
Tests pass on my machine, CLI works as intended as well. Good to go 👍

**Pull request merged**